src/components/common/SceneWrapper.tsx

```diff
@@ -206,19 +204,10 @@ function SceneWrapperInnerComponent(props: SceneWrapperInnerProps) {
     () => ({
       top: safeAreaInsets.top + maybeHeaderHeight,
       right: safeAreaInsets.right,
-      bottom: maybeInsetBottom + maybeNotificationHeight + maybeTabBarHeight + footerHeight,
+      bottom: maybeInsetBottom + maybeNotificationHeight + maybeTabBarHeight,
```

Comment:
This approach will not work. If we convert `insetStyles` and `undoInsetStyles` into animated styles, this will cause big performance problems with the drawer animation.

This is because styles like `bottom` affect layout, which implies a Yoga recalculation. Since we are applying this at the scene level, the yoga update will affect the entire scene, which isn't going to be cheap. Because this is an animation, the calculation needs to happen on every frame.

On the flip side, we definitely need to include the footer height in the insets / undoInsets numbers to get correct rendering.

So, we want the `footerHeight` to be an normal React value, not an animated value.

The problem is that we do *not* want to store the `footerHeight` in the global state. The `footerHeight` is already unique on a scene-by-scene basis. If scene A has a 5 rem footer, and scene B has a 3 rem footer, why does scene A need to re-render when we switch to scene B? This makes no sense.

Instead, the `renderFooter` should be a prop on the `SceneWrapper` component, along with a `footerHeight` property. *IF* the scene wrapper knows that we have a tab bar, then, and only then, do we send the `footerHeight` and `renderFooter` into the tabbar context. The only one subscribed to the tabbar context is the tabbar, so there is very little rendering impact.

Now, you might say that a static `footerHeight` is a bad idea. What if we want to measure our content? It's fine:

```ts
function SomeScene(props) {
  const [footerHeight, setFooterHeight] = useState()
  const handleLayout = useHandler(layoutEvent => setFooterHeight(...))

  const renderFooter = useCallback(() => {
    return <View onLayout={handleLayout}>...</View>
  }, [...])

  return <SceneWrapper
    renderFooter={renderFooter}
    footerHeight={footerHeight}
    >...</SceneWrapper>
}
```

If a scene uses this pattern, then it will render twice - once at mount, and again immediately once the layout updates. However, *only* the affected scene will re-render, and also the tab bar if used.

On the other hand, we already know the height of the search bar in rem's, so there is no need to bother with this on a scene that simply has search.

Reply:
I decided to measure content inline especially for WalletListScene where the footer does change based on the context of whether we're sorting wallets or not. So, since I have that infra on the WalletListScene, I decided to use it on the other scenes as well. I think rendering a scene twice that has a footer should be safe enough considering that we have to re-render for other scene state anyway.

The real expense now is only on initial render for a scene to assigned the scroll handler, which we can try to golf for more performance later, or we can accept this performance hit and optimize other areas.